data/announcements/example.yaml

```diff
@@ -0,0 +1,28 @@
+---
+### Example announcement
+#
+# Leave this example announcement in place. It serves as a template
+# and as documentation by example.
+#
+# 🛈  Announcement changes require approval from @kubernetes/steering-committee
+#
+#
+apiVersion: docs.k8s.io/v1alpha1
+kind: Announcement
+spec:
```

Comment:
does this do anything currently? if not, suggest removing it... it'll just get copy/pasted and not add much value

Reply:
The `example.yaml`? It's an example.

If you mean `.spec` then you can't omit it with the _current_ code. IMO it makes the Hugo layout code a bit nicer because I can `{{ with .spec }}` to bring just those fields into scope.

If you mean the `apiVersion` and `kind` then the current code enforces that you are using this particular format, but we can remove that validation if it's not useful.

Reply:
the apiVersion/kind/spec bits... I didn't see any enforcement there, so it didn't seem like it was accomplishing anything

a flat structure seemed easier to read, but maybe that's just me:

```yaml
# start/stop are required, and must be in format YYYY-MM-DDTHH:MM:SS
start: "2020-01-01T00:00:00"
stop: "2021-04-01T00:00:00"
# message is required
message: |-
  Marklar marklar marklar marklar marklar.
  Marklar marklar.
  [Marklar](https://en.wikipedia.org/wiki/Starvin%27_Marvin_in_Space#Plot)!
# title is optional
title: Sample announcement
# style is optional; if using, set both color and background
style: "color: #fff; background: #000;"
```

Reply:
> 
> 
> does this do anything currently? if not, suggest removing it... it'll just get copy/pasted and not add much value

+1 for removing any unused fields

Reply:
OK, I can try to simplify it.

Reply:
Ah, I did have validation in before but must have left that out.

Reply:
It's too bad that the title and message are not localized. I assume you can use a template in the data file.
I have not thought this through ...

```
[announcement]
[announcement.<name>]
title = "Kubernetes Europe"
message = "Come to Kubecon"
```